discovery-infra/monitoring/process.py

```diff
@@ -0,0 +1,94 @@
+import re
+import json
+import logging
+import collections
+
+REMOVED_FIELDS = [
+    "cluster.image_info_ssh_public_key",
+    "cluster.ssh_public_key",
+    "cluster.image_info.ssh_public_key"
+    "cluster.connectivity_majority_groups",
+    "cluster.controller_logs_collected_at",
+    
+    "cluster.hosts.connectivity",
+    "cluster.hosts.images_status",
+
+    "cluster.image_info_download_url",
+    "cluster.image_info_expires_at",
+    "cluster.image_info_size_bytes",
+    "cluster.ingress_vip",
+    "link"
+]
+
+SKIPPABLE_EVENTS = [
+    "reached installation stage Writing image to disk"
+]
+
+logging.basicConfig(level=logging.WARN, format='%(levelname)-10s %(message)s')
+logger = logging.getLogger(__name__)
+logging.getLogger("__main__").setLevel(logging.INFO)
+
+
+def convert_field_to_json(converted_field):
```

Comment:
this is used only here no?

Reply:
yes, but it is not part of the class functionality so its not really an attribute of the class